src/components/ItemDetail/ItemDetailMetadata.js

```diff
@@ -1,23 +1,83 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import LineEntry from './LineEntry';
+import MultiMetadata from './MultiMetadata';
+import SingleMetadata from './SingleMetadata';
 
 const ItemDetailMetadata = props => {
   if (!props.item) {
     return [];
   }
-  console.log('Props: ' + props);
 
-  const { title } = props.item;
+  const {
+    title = null,
+    abstract = null,
+    caption = null,
+    contributor = null,
+    date = null,
+    description = null,
+    admin_set = null, // division
+    keyword = null,
+    language = null,
+    location = null,
+    provenance = null,
+    publisher = null,
+    related_url = null,
+    rights_holder = null,
+    source = null,
+    subject = null,
+    rights_statement = null,
+    extra_fields = null
+  } = props.item;
+
+  let getCreators = contributors => {
+    let creators = [];
+    for (let entry of contributors) {
+      if (entry.type === 'creator') creators.push(entry);
```

Comment:
Run this through prettier and see if it allows `if` blocks on the same line.  I personally don't like them, but we'll let Prettier be the boss.

Also, if you feel like trying something new, use `Array.filter()` here instead of the `for...of`.  But good to see `for... of` instead of the old skool way!